RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
just a public method that works, no other considerations.
Making edits in the global object that may be loaded by others looks like a worse idea.
deepcopy may be better
@Dr15Jones please clarify if VPSet::copy method is safe